vector/src/main/res/values/theme_dark.xml

```diff
@@ -8,23 +8,24 @@
         <item name="colorPrimary">@color/primary_color_dark</item>
         <item name="colorAccent">@color/accent_color_dark</item>
 
-        <item name="android:textColorPrimary">@color/riot_primary_text_color_dark</item>
-        <item name="android:textColorSecondary">@color/riot_secondary_text_color_dark</item>
+        <item name="android:textColorPrimary">@color/riot_primary_text_color_black</item>
```

Comment:
no "_black" color in dark theme please. Do not hesitate to create color alias (also for lines below)